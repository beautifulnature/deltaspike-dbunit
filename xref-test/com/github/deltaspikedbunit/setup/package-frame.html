
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "DTD/xhtml1-transitional.dtd">
<html xml:lang="en" lang="en">
	<head>
		<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
		<title>Deltaspike DBUnit 0.5.0 Reference Package com.github.deltaspikedbunit.setup</title>
		<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="style" />
	</head>
	<body>

		<h3>
        	<a href="package-summary.html" target="classFrame">com.github.deltaspikedbunit.setup</a>
      	</h3>

      	<h3>Classes</h3>

      	<ul>
      		          	<li>
            	<a href="CleanInsertSetupOnClassTest.html" target="classFrame">CleanInsertSetupOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="CleanInsertSetupOnMethodTest.html" target="classFrame">CleanInsertSetupOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="DeleteAllSetupOnClassTest.html" target="classFrame">DeleteAllSetupOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="DeleteAllSetupOnMethodTest.html" target="classFrame">DeleteAllSetupOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="DeleteSetupOnClassTest.html" target="classFrame">DeleteSetupOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="DeleteSetupOnMethodTest.html" target="classFrame">DeleteSetupOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="InsertSetupOnClassTest.html" target="classFrame">InsertSetupOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="InsertSetupOnMethodTest.html" target="classFrame">InsertSetupOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="MixedSetupOnClassAndMethodTest.html" target="classFrame">MixedSetupOnClassAndMethodTest</a>
          	</li>
          	          	<li>
            	<a href="MultipleInsertSetupOnClassTest.html" target="classFrame">MultipleInsertSetupOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="RefreshSetupOnClassTest.html" target="classFrame">RefreshSetupOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="RefreshSetupOnMethodTest.html" target="classFrame">RefreshSetupOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="TruncateSetupOnClassTest.html" target="classFrame">TruncateSetupOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="TruncateSetupOnMethodTest.html" target="classFrame">TruncateSetupOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="UpdateSetupOnClassTest.html" target="classFrame">UpdateSetupOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="UpdateSetupOnMethodTest.html" target="classFrame">UpdateSetupOnMethodTest</a>
          	</li>
          	      	</ul>

	</body>
</html>