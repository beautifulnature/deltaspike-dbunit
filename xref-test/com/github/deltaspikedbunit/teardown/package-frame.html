
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "DTD/xhtml1-transitional.dtd">
<html xml:lang="en" lang="en">
	<head>
		<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
		<title>Deltaspike DBUnit 0.5.0 Reference Package com.github.deltaspikedbunit.teardown</title>
		<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="style" />
	</head>
	<body>

		<h3>
        	<a href="package-summary.html" target="classFrame">com.github.deltaspikedbunit.teardown</a>
      	</h3>

      	<h3>Classes</h3>

      	<ul>
      		          	<li>
            	<a href="CleanInsertTearDownOnClassTest.html" target="classFrame">CleanInsertTearDownOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="CleanInsertTearDownOnMethodTest.html" target="classFrame">CleanInsertTearDownOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="DeleteAllTearDownOnClassTest.html" target="classFrame">DeleteAllTearDownOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="DeleteAllTearDownOnMethodTest.html" target="classFrame">DeleteAllTearDownOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="DeleteTearDownOnClassTest.html" target="classFrame">DeleteTearDownOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="DeleteTearDownOnMethodTest.html" target="classFrame">DeleteTearDownOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="InsertTearDownOnClassTest.html" target="classFrame">InsertTearDownOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="InsertTearDownOnMethodTest.html" target="classFrame">InsertTearDownOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="MixedTearDownOnClassAndMethodTest.html" target="classFrame">MixedTearDownOnClassAndMethodTest</a>
          	</li>
          	          	<li>
            	<a href="MultipleInsertTearDownOnClassTest.html" target="classFrame">MultipleInsertTearDownOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="RefreshTearDownOnClassTest.html" target="classFrame">RefreshTearDownOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="RefreshTearDownOnMethodTest.html" target="classFrame">RefreshTearDownOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="TruncateTearDownOnClassTest.html" target="classFrame">TruncateTearDownOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="TruncateTearDownOnMethodTest.html" target="classFrame">TruncateTearDownOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="UpdateTearDownOnClassTest.html" target="classFrame">UpdateTearDownOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="UpdateTearDownOnMethodTest.html" target="classFrame">UpdateTearDownOnMethodTest</a>
          	</li>
          	      	</ul>

	</body>
</html>