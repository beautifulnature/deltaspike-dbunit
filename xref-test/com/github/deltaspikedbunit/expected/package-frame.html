
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "DTD/xhtml1-transitional.dtd">
<html xml:lang="en" lang="en">
	<head>
		<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
		<title>Deltaspike DBUnit 0.5.0 Reference Package com.github.deltaspikedbunit.expected</title>
		<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="style" />
	</head>
	<body>

		<h3>
        	<a href="package-summary.html" target="classFrame">com.github.deltaspikedbunit.expected</a>
      	</h3>

      	<h3>Classes</h3>

      	<ul>
      		          	<li>
            	<a href="ExpectedFailureOnClassTest.html" target="classFrame">ExpectedFailureOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedFailureOnMethodTest.html" target="classFrame">ExpectedFailureOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedNonStrictOnClassTest.html" target="classFrame">ExpectedNonStrictOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedNonStrictOnMethodTest.html" target="classFrame">ExpectedNonStrictOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedNonStrictUnorderedOnClassTest.html" target="classFrame">ExpectedNonStrictUnorderedOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedNonStrictUnorderedOnMethodTest.html" target="classFrame">ExpectedNonStrictUnorderedOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedOnClassAndMethodTest.html" target="classFrame">ExpectedOnClassAndMethodTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedOnClassAndMethodWithoutOverrideTest.html" target="classFrame">ExpectedOnClassAndMethodWithoutOverrideTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedQueryNonStrictOnMethodTest.html" target="classFrame">ExpectedQueryNonStrictOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedQueryOnMethodTest.html" target="classFrame">ExpectedQueryOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedQueryWithModifierOnMethodTest.html" target="classFrame">ExpectedQueryWithModifierOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedTableNonStrictOnClassTest.html" target="classFrame">ExpectedTableNonStrictOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedQueryWithModifierOnMethodTest.html" target="classFrame">InnerModifier</a>
          	</li>
          	          	<li>
            	<a href="ExpectedQueryWithModifierOnMethodTest.html" target="classFrame">OuterModifier</a>
          	</li>
          	      	</ul>

	</body>
</html>